release-team/role-handbooks/release-notes/README.md

```diff
@@ -105,6 +105,7 @@ Release Notes lead should be responsible for granting team members required acce
 1. Add OWNERS file into release notes directory. [Sample PR](https://github.com/kubernetes/sig-release/pull/1943) for v1.25 release.
 2. Add release notes team members for CHANGELOG review/approval.
    [Sample PR](https://github.com/kubernetes/kubernetes/pull/112045) for v1.25 release.
+3. Add release notes team members to the Kubernetes org as a GitHub team [Sample PR](https://github.com/kubernetes/org/pull/3735)
```

Comment:
At the start of the cycle the release-lead updates the document - so I would rephrase this a little bit.


```suggestion
3. Typically, the release lead updates the [release notes team members](https://github.com/kubernetes/org/blob/main/config/kubernetes/sig-release/teams.yaml#L387) along with other teams in a single PR; check the PR and make sure the release notes team has been updated, otherwise open a PR yourself [sample PR](https://github.com/kubernetes/org/pull/3735)
```